contents.json

```diff
@@ -737,7 +737,12 @@
 		"category": "ai",
 		"description": "The iOS and OS X neural network framework",
 		"homepage": "https://github.com/aleph7/BrainCore"
-	}, {
+	},{
+		"title": "MLKit",
+		"category": "ai",
+		"description": "A simple machine learning framework written in Swift 🤖",
```

Comment:
we're trying to clean up descriptions.. can you remove written in Swift and the emoji

thanks!